packages/react/src/Pagehead/Pagehead.types.test.tsx

```diff
@@ -6,6 +6,5 @@ export function shouldAcceptCallWithNoProps() {
 }
 
 export function shouldNotAcceptSystemProps() {
-  // @ts-expect-error system props should not be accepted
```

Comment:
Torn on this change, cause it now will technically accept the system props we were blocking before. I think the change comes from the toggleStyledComponent somewhere. But this is temporary. When we remove the feature flag it will be an error again.

Reply:
I can hold off on this and try and find a solution if you like? I don't mind, the temporary change could be in place for awhile depending on how things go?

Reply:
Na, if we work fast enough on this, it won't make a difference. and I don't think folks are adding these attributes in dotcom much anymore. if at all